R_program/Main.R

```diff
@@ -574,12 +578,117 @@ remove(temp)
 
 ############################# End MARKOV Weather Generator Code ##############################
 
+############################# Phenology Code ###############################
+# This code determines plant functional type phenology
+# and then scales STEPWAT2 parameters accordingly
+if(rescale_phenology){
+  source(vegetation.file)
+  setwd(db_loc)
+  growingSeason <- read.csv("InputData_GrowingSeason.csv", header = TRUE);
+  
+  # Parse growingSeason for phenology values
+  phenology <- growingSeason[grepl(".*_phenology", growingSeason[ , 1]), , ]
+  # Use regex to name the rows using the capture group (.*)
```

Comment:
perhaps clearer just to say:

#Set the rgroup names as rownames of object phenology

same for biomass and litter below.

Reply:
Resolved by commit f15f202.